Signal/src/ViewControllers/InboxTableViewCell.m

```diff
@@ -49,41 +50,56 @@ - (nullable NSString *)reuseIdentifier
     return NSStringFromClass(self.class);
 }
 
-- (void)configureWithThread:(TSThread *)thread contactsManager:(OWSContactsManager *)contactsManager
+- (void)configureWithThread:(TSThread *)thread
+            contactsManager:(OWSContactsManager *)contactsManager
+      blockedPhoneNumberSet:(NSSet<NSString *> *)blockedPhoneNumberSet;
 {
-    if (!_threadId || ![_threadId isEqualToString:thread.uniqueId]) {
-        dispatch_async(dispatch_get_main_queue(), ^{
```

Comment:
Oh, I'm wrong.  We _are_ caching avatars inside `OWSContactAvatarBuilder` using contactsManager.avatarCache.  I wonder if we need the to do anything off the main thread. I'll try to test on a 5c.